### DISCUSSION TITLE
Installing SSL to Chaskiq docker

### DISCUSSION THREAD

**Discussion by Sypher1k** [chaskiq/chaskiq#263] [Q&A] [answered]
Does anyone know how to add ssl certicate to Chaskiq on docker? 
I followed [this guide](https://dev.chaskiq.io/en/articles/docker-for-development)  and successfully installed Chaskiq on my aws ec2 ubuntu 20.*  server. Docker is running on port 3000.

**Comment by michelson** [comment, marked as answer]
That guide is for development, check the production guides instead.
For production environments you usually put the certificate on a load balancer (or a machine with nginx), and the load balancer will handle the certificate and point to your internal port (3000 in your case), also if you deploy the app with caprover you will have ssl support out of the box.

**Reply by Sypher1k** [reply]
I got it working by using Nginx. Thank you!!

**Reply by Bianconiglio** [reply]
Hi i have set .env and nginx but for me dont work the page is blank 

.env

HOST=http://xyz.mydomain.com:3000
ASSET_HOST=http://xyz.mydomain.com:3000
WS=ws://xyz.mydomain.com:3000/cable

nginx

server {
        listen 80 default_server;
        listen [::]:80 default_server;

	server_name xyz.mydomain.com;
	listen 443 ssl; # managed by Certbot

	ssl_certificate /etc/letsencrypt/live/xyz.mydomain.com/fullchain.pem; # managed by Certbot
    	ssl_certificate_key /etc/letsencrypt/live/xyz.mydomain.com/privkey.pem; # managed by Certbot



 	location / {


	 	proxy_pass http://localhost:3000;
 	}
 }

do you can help me? if use nginx and pot 80 work good

**Reply by michelson** [reply]
.env file is not for production. Also, SSL cant be accessed through port 3000.

**Reply by Bianconiglio** [reply]
Thank you very much for your reply, do you have a production .env sample file that you can see for example?

**Reply by michelson** [reply]
as I said, .env file is not for production. in a production environment, you need to set up env vars differently. The variable names should be the same. in your case the url should not have the :3000 port and the ws:// protocol should be wss://

**Reply by Bianconiglio** [reply]
Thank you so much for your help now work!
